src/components/ProxyDetails/ProxyConnect.vue

```diff
@@ -18,8 +18,9 @@ const showProxyButton = asyncComputed(() => extension.isAllowedIncognitoAccess()
     <ol class="ml-4 list-decimal">
       <li>
         <p class="flex items-center">
-          Click the&nbsp;<em class="inline-flex"><IcRoundMenu /></em>&nbsp;icon in the top right
-          corner of the browser
+          Click the
+          <FeMenu />
```

Comment:
You need a non-breaking space between `the` and the icon.
<img width="120" alt="image" src="https://user-images.githubusercontent.com/1008390/224696317-1af4a60c-5cd3-45a8-95b9-dc702f0a4974.png">

Reply:
Fixed.